src/app/static/src/app/components/adr/SelectDatasetModal.vue

```diff
@@ -49,7 +49,7 @@
                     :disabled="disableImport"
                     class="btn btn-red"
                     v-translate="'import'"
-                    @click="$emit('confirmImport', datasetId, datasetReleaseId)">
+                    @click="$emit('confirmImport', datasetId!!, datasetReleaseId)">
```

Comment:
this button is disabled (`disableImport`) if `datasetId` doesn't exist anyway so asserting that `datasetId` is truthy